clients/lsp-rust.el

```diff
@@ -495,6 +495,12 @@ The command should include `--message=format=json` or similar option."
   :group 'lsp-rust-analyzer
   :package-version '(lsp-mode . "6.2.2"))
 
+(defcustom lsp-rust-analyzer-cargo-all-targets t
+  "Cargo watch all targets or not."
+  :type 'boolean
+  :group 'lsp-rust-analyzer
+  :package-version '(lsp-mode . "6.2.2"))
```

Comment:
The version should be 8.0.2.

Reply:
Even if it's a revert? The flag only disappeared for a month or so.

If it counts as a new flag then I'd like to change the name to `lsp-rust-analyzer-check-all-targets` to match the new naming of rust-analyzer.

Reply:
> If it counts as a new flag then I'd like to change the name to lsp-rust-analyzer-check-all-targets to match the new naming of rust-analyzer.

Probably better.

Reply:
Sounds good. Done.